ui-particles-angular/projects/ui-particles-angular/src/lib/public-api.ts

```diff
@@ -52,3 +52,7 @@ export * from './gio-avatar';
 export * from './gio-breadcrumb';
 
 export * from './gio-mat-config';
+
+export * from './gio-form-json-schema/gio-form-json-schema.component';
+export * from './gio-form-json-schema/gio-form-json-schema.module';
+export * from './gio-form-json-schema/model/FormlyJSONSchema7';
```

Comment:
WDYT to create a GIoJsonSchema ? and not expose the Formly one ? 
For now it's the same but later we can add some custom stuff and add adapter inside ui-P ?

Reply:
Yes, good idea !